components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/authz/OAuth2AuthzEndpoint.java

```diff
@@ -2411,6 +2413,10 @@ private void handleOIDCRequestObject(OAuthMessage oAuthMessage, OAuthAuthzReques
     private void validateRequestObjectParams(OAuthAuthzRequest oauthRequest) throws RequestObjectException {
 
         // With in the same request it can not be used both request parameter and request_uri parameter.
+        if (Boolean.parseBoolean(oauthRequest.getParam(
```

Comment:
The addition sort of contradicts the comment above it. Add a proper comment mentioning what is being done here.

Reply:
Will be addressed as improvements with the next PR